src/retractions.jl

```diff
@@ -167,26 +168,25 @@ See also [`retract!`](@ref).
 function inverse_retract!(M::AbstractManifold, X, p, q)
     return inverse_retract!(M, X, p, q, default_inverse_retraction_method(M))
 end
-function inverse_retract!(
-    M::AbstractManifold,
-    X,
-    p,
-    q,
-    method::LogarithmicInverseRetraction,
-)
+function inverse_retract!(M::AbstractManifold, X, p, q, ::LogarithmicInverseRetraction)
     return log!(M, X, p, q)
 end
-function inverse_retract!(
-    M::AbstractManifold,
-    X,
-    p,
-    q,
-    method::AbstractInverseRetractionMethod,
-)
-    return error(
-        manifold_function_not_implemented_message(M, inverse_retract!, X, p, q, method),
-    )
+
+#
+# dispatch to lower level
+function inverse_retract!(M::AbstractManifold, X, p, q, ::PolarInverseRetraction)
+    return inverse_retract_polar!(M, X, p, q)
+end
+function inverse_retract!(M::AbstractManifold, X, p, q, ::ProjectionInverseRetraction)
+    return inverse_retract_porect(M, X, p, q)
 end
+function inverse_retract!(M::AbstractManifold, X, p, q, ::QRInverseRetraction)
+    return inverse_retract_qr(M, X, p, q)
+end
+function inverse_retract!(M::AbstractManifold, X, p, q, m::NLsolveInverseRetraction)
+    return inverse_retract_ode(M, X, p, q, m)
```

Comment:
```suggestion
    return inverse_retract_nlsolve!(M, X, p, q, m)
```
IIRC this doesn't use any ODEs. Also, why do we need it in ManifoldsBase.jl if it's not implemented in this package? This isn't really something people would extend.

Reply:
Ah I confused it with something else, sure. I am not sure why this is in base...